src/views/examples/Profile.js

```diff
@@ -34,20 +34,50 @@ import {
 
 // core components
 import UserHeader from "components/Headers/UserHeader.js";
+import axios from 'axios';
 
-const Profile = (props) => {
+const Profile = () => {
 
-  const [resumelink,setResumelink]=useState(props.resumeLink);
-  const [githublink,setGithublink]=useState(props.githubLink);
-  const [linkedinlink,setLinkedinlink]=useState(props.linkedinLink);
+  const [profiledata,setProfileData]=useState([]);
 
-  const printdata = (event)=>{
-    console.log({resumelink},{githublink},{linkedinlink});
-    event.preventDefault();
+  const fetchProfile = async() => {
+    const profileDetails= await axios.get('http://127.0.0.1:8000/user/profile/')
+    setProfileData(profileDetails.data)
+    setMasterResumelink(profiledata.mastercv)
+    setResume1link(profiledata.resume1)
+    setResume2link(profiledata.resume1)
+    setGithublink(profiledata.github)
+    setLinkedinlink(profiledata.linkedin)
+
```

Comment:
Call this just after user login as it not efficient way. You should use redux to store these value and then use these here.